DX/Docker/Wrapper.md

```diff
@@ -0,0 +1,50 @@
+# v2: Dockerized Deployment Wrapper
+
+## Objective
+
+To facilitate common tasks performed with eZ Platform and its entourage of
+technologies, a command-line wrapper for frequently performed activities is
+desirable. Given that Docker unifies Development, Staging, and Production
+environments, such tooling will be used to unify development environments as
+well using Docker. The solution will be "aware" of the difference between
+running on a clustered configuration, and provide assistance to the developer in
+issuing relevant commands to manage common tasks in this arena.
+
+Deliverable: The solution shall be one _command_, with options, documented with
+instructions, runnable on Unix, Linux, Mac, and Windows development, staging,
+and production machines, enabling a single approach to common tasks regardless
+of environment.
+
+## Wrapper Solution Requirements
+
+### Firm
+The solution MUST:
+1. Run _inside_ of a Docker container, to unify environments across all systems,
```

Comment:
It seems we have here an issue. On one end, we have the opportunity to build a PHP tool that would deliver most of what we try to achieve, but add the requirement of having PHP running on the command line of the developer environment. On the other, we can still look for a full Docker based solution, but it seems we don't really know how to get there, nor what would be the effort or the technical solution. Pragmatically, I'd like to go for the first route, to quickly enable developers to deploy on their machines. I'd like to revise this requirement expect if we can show a fully Docker based solution that is requiring same level of effort/risk. I have looked at other applications available on Docker hub (such as Drupal or Wordpress) but each time, for anything complex, they end up asking the user to manually do Docker Compose, so I'm not sure about the value.

My vote: be pragmatic and go for a 1st step which require Docker and PHP on CLI.
Insights here highly appreciated.